config/environments/production.rb

```diff
@@ -4,6 +4,11 @@
   # Settings specified here will take precedence over those in
   # config/application.rb.
 
+  config.hosts = [
+    "coronavirus-business-volunteers.service.gov.uk",
+    "govuk-coronavirus-business-volunteer-form-stg.cloudapps.digital",
```

Comment:
There's also d2uxybyx2btt89.cloudfront.net, which is the CDN fronted domain for staging.